test/e2e/policies.test-suite/topology-aware/nri-resmgr.cfg

```diff
@@ -0,0 +1,8 @@
+policy:
+  Active: topology-aware
+  ReservedResources:
+    CPU: 750m
+logger:
+  Debug: nri-resmgr,resource-manager,cache,policy
```

Comment:
Should we have message dumping (`nri-plugin` debug flag) enabled for the tests to help any post mortem analysis in case of failures ? If we do, should we have the same for `balloons` tests ?